### PR TITLE
Execution metadata

### PR DESCRIPTION
**Pull request by apryka**
## Related tickets?

https://github.com/keen/keen-dataviz.js/issues/135

## How should this be tested?

Step through the code line by line. Things to keep in mind as you review:
 - Are there any edge cases not covered by this code?
 - Does this code follow conventions (naming, formatting, modularization, etc) where applicable?

Fetch the branch and/or deploy to staging to test the following:

- [ ] Does the code compile without warnings (check shell, console)?
- [ ] Do all tests pass?
- [ ] Does the UI, pixel by pixel, look exactly as expected (check various screen sizes, including mobile)?
- [ ] If the feature makes requests from the browser, inspect them in the Web Inspector. Do they look as expected (parameters, headers, etc)?
- [ ] If the feature sends data to Keen, is the data visible in the project if you run an extraction (include link to collection/query)?
- [ ] If the feature saves data to a database, can you confirm the data is indeed created in the database?

### CODE REVIEW

File: lib/style/keen-dataviz.css [hunk 1]
```diff
@@ -298,6 +298,30 @@
...
+  &-execution-meta {
+    padding: 0.5rem 1rem;
+    margin: 0.75rem 0;
+    display: inline-grid;
+    grid-template-columns: repeat(2, max-content);
+    grid-gap: 0.5em 1em;
+    text-transform: uppercase;
+    font-size: 0.75rem;
+    border: 1px solid #eee;
+
```
Comment (adamkasprowicz) [inline lib/style/keen-dataviz.css]: please add
```css
    position: absolute;
    z-index: 999999;
    background: #fff;
```

File: lib/utils/render-execution-metadata.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+  if (Object.keys(data).length) {
+    Object.keys(data).forEach((item) => {
+      const dt = document.createElement('dt');
+      dt.innerText = item.replace(/_/g, ' ');
+      const dd = document.createElement('dd');
+      dd.innerText = data[item];
+
+      dataList.append(dt);
+      dataList.append(dd);
```
Comment (adamkasprowicz) [inline lib/utils/render-execution-metadata.js]: + copy to clipboard
